sys/arm64/arm64/pmap.c

```diff
@@ -5189,14 +5233,16 @@ pmap_remove_pages(pmap_t pmap)
 				/*
 				 * Update the vm_page_t clean/reference bits.
 				 */
-				if (pmap_pte_dirty(pmap, tpte)) {
+				if (pmap_pte_dirty(pmap, tpte) ||
+				    pmap_pte_capdirty(pmap, tpte)) {
```

Comment:
Does this condition want pushing into the lvl == 1 case?

Reply:
Happy to do this, it would save an extra check in the non-block case.